### PR TITLE
#232F Start message expression

### PR DESCRIPTION
**Pull request by CarlosNZ**
Update to mock data so changes in [front-end #242](https://github.com/openmsupply/application-manager-web-app/pull/242) can be demonstrated.

~~**NOTE**: This still uses the old array format for the evaluator objects. It will need to be updated after #159 is merged (and again after #158), so hopefully they can be approved before this one and I'll merge them back into here and do those changes before this is merged.~~
(Done)

### CODE REVIEW

File: database/insertData.js [hunk 1]
```diff
@@ -13,7 +13,28 @@ const queries = [
           code: "TestRego"
           name: "Test -- General Registration"
           isLinear: false
-          startMessage: "## This is the general registration for feature showcase\\n- Proof of identity (Passport, Drivers license)\\n- Proof of your medical certification\\n- Drug ingredient list\\n- Product images\\n- Packging images"
+          startMessage: {
```
Comment (nmadruga) [inline database/insertData.js:16]: Would this be a good place to use the `substring` evaluator from PR [#158](https://github.com/openmsupply/application-manager-server/pull/164)?

**Review by nmadruga — CHANGES_REQUESTED** [review]
Looks good and it's displaying in the Start page correctly.

Just 1 comment to address please.

File: database/insertData.js (hunk 1 above)
Comment (CarlosNZ) [reply database/insertData.js:16]: Yes, I was planning to, but can't yet as that update has not been published to package yet. 

I'll need to do a new PR to update the version in package.JSON once the evaluator has been updated, and was planning to change these strings in that PR.

Comment (nmadruga) [reply database/insertData.js:16]: Ok. No problem. I was just pointing out because it seems more redundant using the `substring` type

**Review by nmadruga — APPROVED** [review]

**Pull request merged**